src/enclave/enclave.h

```diff
@@ -127,8 +127,9 @@ namespace enclave
       return true;
     }
 
-    bool run()
+    bool run_main()
     {
+      LOG_INFO << "Running main thread" << std::endl;
```

Comment:
All the logging should be `LOG_INFO_FMT("Running main thread")`. We may want to have them as `LOG_DEBUG_FMT()`?